crates/autopilot/src/domain/settlement/transaction.rs

```diff
@@ -0,0 +1,3 @@
+/// Call data in a format expected by the settlement contract.
+#[derive(Debug)]
+pub struct CallData(pub crate::util::Bytes<Vec<u8>>);
```

Comment:
I would move this abstraction to the `eth` module (also not sure if wrapping is an advantage or it should just be a type alias)

Reply:
fixed